bower.json

```diff
@@ -14,6 +14,7 @@
     "dist/angular-leaflet-directive.js"
   ],
   "dependencies": {
+    "Leaflet.vector-markers": "https://github.com/hiasinho/Leaflet.vector-markers",
```

Comment:
@molehillrocker Although, now that I checked, that package already has release tags (latest 0.0.3) and it's also at Bower:
```bash
$ bower search vector-markers
Search results:

    leaflet.vector-markers git://github.com/hiasinho/Leaflet.vector-markers.git
```